website/constants.py

```diff
@@ -0,0 +1,13 @@
+from os import environ
+from typing import NamedTuple
+
+import dotenv
+
+dotenv.load_dotenv()
+
+REDIRECT_URI = environ.get("REDIRECT_URI")
```

Comment:
Is there a reason we're separating out the client namedtuple like this? And not constructing an instance? What purpose does this serve except for categorization? I'm not sure what benefit pydantic would have here, but I don't think we should be using Client as a class just for namespacing. CLIENT_ID and CLIENT_SECRET are fine constants

Reply:
> Is there a reason we're separating out the client namedtuple like this? And not constructing an instance? What purpose does this serve except for categorization? I'm not sure what benefit pydantic would have here, but I don't think we should be using Client as a class just for namespacing. CLIENT_ID and CLIENT_SECRET are fine constants

I kept the "client" side constants inside the namedtuple to keep them as a group.

Reply:
Well, I think it's not worth the effort at this time to group them. However it doesn't matter that much. But they shouldn't be a NamedTuple. We're never constructing it, so it should just be a bare class to reduce imports and potential confusion. But it'd be better without grouping